src/Components/Blueprints/ImportBlueprintModal.test.tsx

```diff
@@ -374,6 +374,9 @@ describe('Import modal', () => {
     );
     expect(keyboardDropDown).toHaveValue('us');
 
+    // Hostname
     await clickNext();
+    const hostnameInput = await screen.findByPlaceholderText(/Add a hostname/i);
+    expect(hostnameInput).toHaveValue('baseimage');
```

Comment:
what do you think about 
```suggestion
    expect(hostnameInput).toHaveValue('base-image');
```

Reply:
Sure, updated.

Reply:
great, I think you can update this as well, it is not coming from that or, but for consistency :)
[customizations]
hostname = "baseimage"
fips = true